include/boost/math/distributions/logistic.hpp

```diff
@@ -221,15 +219,8 @@ namespace boost { namespace math {
        {
           return policies::raise_overflow_error<RealType>(function,"probability argument is 1, must be >0 and <1",Policy());
        }
-       //Expressions to try
-       //return location+scale*log(p/(1-p));
-       //return location+scale*log1p((2*p-1)/(1-p));
-
-       //return location - scale*log( (1-p)/p);
-       //return location - scale*log1p((1-2*p)/p);
 
-       //return -scale*log(1/p-1) + location;
-       return location - scale * log((1 - p) / p);
+       return location - scale * -logit(p, Policy());
```

Comment:
Another very minor nitpick: remove the double negative, so the expression is `location + scale * logit(p, Policy())`.  That's the usual expression for going from a 'standard' value to a value in the `(location, scale)` family.  I assume the original `-` in front of `scale` was there because `log((1 - p)/p)` was used instead of `log(p/(1 - p))`.